Code/GraphMol/QueryOps.h

```diff
@@ -645,6 +647,14 @@ T *makeAtomNonHydrogenDegreeQuery(int what, const std::string &descr) {
 RDKIT_GRAPHMOL_EXPORT ATOM_EQUALS_QUERY *makeAtomNonHydrogenDegreeQuery(
     int what);
 
+//! returns a Query for matching bridgehead atoms
+template <class T>
+T *makeAtomIsBridgeheadQuery(const std::string &descr) {
+  return makeAtomSimpleQuery<T>(true, queryIsAtomBridgehead, descr);
+}
+//! \overload
+RDKIT_GRAPHMOL_EXPORT ATOM_EQUALS_QUERY *makeAtomIsBridgeheadQuery();
```

Comment:
I think the failed test is happening because this isn't defined.

Reply:
yeah, I forgot to make that change.